content/ja/docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -80,7 +80,7 @@ selector](/ja/docs/concepts/scheduling-eviction/assign-pod-node/)にマッチす
 
 ### デフォルトスケジューラーによってスケジューリングされる場合
 
-{{< feature-state state="stable" for-kubernetes-version="1.17" >}}
+{{< feature-state for_kubernetes_version="1.17" state="stable" >}}
```

Comment:
```suggestion
{{< feature-state for_k8s_version="1.17" state="stable" >}}
```
Is valid property name `for_k8s_version`, not `for_kubernetes_version`?

I could found `for_k8s_version` around other doc.
e.g. [here](https://github.com/kubernetes/website/blob/main/content/en/docs/concepts/workloads/controllers/job.md#completion-mode)

Reply:
and you can find other place which using `for_k8s_version` property by command following:
`grep -r "for_k8s_version"`

Reply:
Thank you to comment.

I created [PR which fix this topic, and merged](https://github.com/kubernetes/website/pull/33071).
Could you update this PR or close this and create other PR?